config/jobs/kubernetes-sigs/ingress-controller-conformance/ingress-controller-conformance.yaml

```diff
@@ -76,7 +76,7 @@ presubmits:
         args:
         - bash
         - -c
-        - pip3 install reformat-gherkin==2.0.0 --ignore-installed PyYAML && make verify-gherkin
+        - pip3 install reformat-gherkin==3.0.1 --ignore-installed PyYAML && make verify-gherkin
```

Comment:
I recommend moving something like this into the source repo, so you can test changes with a PR, if this change doesn't work you'll break all PR testing and have to come back and revert here.

Reply:
something like
```
args:
- make
- install-ci-deps
- verify-gherkin
```

or a wrapper script `hack/ci-verify-gherkin` or whatever. if possible it will be a better contributor experience to have `make verify-gherkin` just work by setting up a venv and ensuring deps or something but that's a big more involved. to start I'd just move this logic to the source repo.

Reply:
I created https://github.com/kubernetes-sigs/ingress-controller-conformance/pull/109 to move the logic to source repo, once that is in, I will make the change here

Reply:
https://github.com/kubernetes-sigs/ingress-controller-conformance/pull/109 is now merged, so I updated this file to point to a shell script in source repo that handles the installation of reformat-gherkin